app/assets/stylesheets/styles.css.scss

```diff
@@ -0,0 +1,45 @@
+
+body {
+  background: transparent url("texture.png");
+}
+
+#nav_bar {
```

Comment:
Avoid using id selectors<br>Selector `nav_bar` should be written in lowercase with hyphens